torch2trt/converters/stack.py

```diff
@@ -23,6 +27,7 @@ def convert_cat_trt7(ctx):
     layer.axis = dim - 1
```

Comment:
Just to confirm -- this will be invalid if converting `torch.cat(..., 0)` (ie. if we try to cat in the first dimension); however, because we expect that the first dimension is always the batch dimension, we accept this limitation, correct? (Note that this PR with the negative dims preserves this behavior).